kotlinlib/src/mill/kotlinlib/android/AndroidKotlinModule.scala

```diff
@@ -8,6 +8,23 @@ import mill.kotlinlib.{Dep, DepSyntax, KotlinModule}
 // https://kotlinlang.org/docs/compose-compiler-options.html possible options
 trait AndroidKotlinModule extends KotlinModule {
 
+  override def kotlincOptions = super.kotlincOptions() ++ {
+    if (androidEnableCompose()) {
+      Seq(
+        // TODO expose Compose configuration options
+        // https://kotlinlang.org/docs/compose-compiler-options.html possible options
+        s"-Xplugin=${kotlincPluginJars().map(_.path).mkString(",")}",
+        "-P",
+        "plugin:androidx.compose.compiler.plugins.kotlin:generateFunctionKeyMetaClasses=false," +
+          "plugin:androidx.compose.compiler.plugins.kotlin:traceMarkersEnabled=true",
+        "-verbose",
+        "-opt-in=kotlin.RequiresOptIn",
+        "-opt-in=kotlin.Experimental",
+        "-Xallow-unstable-dependencies"
+      )
+    } else Seq.empty
+  }
```

Comment:
Something is fishy here. We add a plugin but we don't bring it it. We already set up inherited `kotlincPluginJars` was most likely applied already.